### ISSUE TITLE
Temp font size

### ISSUE THREAD

**Issue by tampiss** [ilcato/homebridge-Fibaro-HC2#121]
Hello, is there a chance to round the temp from HC2 to Homekit Home app widgets to whole numbers (23,4 to 23)? Due to having decimal numbers, there are so small to read within the widgets in Home app. Thank you, M

**Comment by lboue** [comment]
Hello,

Are you sure it will work? On my iPhone, the zero after the decimal point always appears. You can downgrade to iOS11 as a workaround.

Regards

**Comment by tampiss** [comment]
Yes, Im on iOS 12. It display as you say 12,0... Id rather 12... To have the font bigger. Im ok with iOS 12... do not want to downgrad...

**Comment by tampiss** [comment]
Well nothing special, just the fact that it has decimals, the font is unreadable. But I got used to it, so no worries. M

**Comment by ilcato** [comment]
Unreadable on which device?

**Comment by tampiss** [comment]
Well see the print-screen https://photos.app.goo.gl/bcpTiBE87CHT2hwG8 its on iphone x. M
